_posts/en/newsletters/2025-01-17-newsletter.md

```diff
@@ -0,0 +1,153 @@
+  via LN).  David Caseria [replied][caseria pplns] that in some _pay per
+  last N shares_ ([PPLNS][topic pplns]) schemes, such as
+  [TIDES][recap291 tides], a miner might need to wait for the pool to
+  find several blocks, which might take days or weeks for a small pool.
+  Instead of waiting, a miner with ecash shares could immediately sell
+  them on an open market (without disclosing to the pool or any third
+  party anything about their identity, not even any ephemeral identity
+  they used when mining).
+
+  Caseria also noted that existing mining pools find it financially
+  challenging to support the _full paid per share_ ([FPPS][topic fpps])
+  scheme where a miner is paid proportional to the entire block reward
+  (subsidy plus transaction fees) when they create a share.  He didn't
+  elaborate, but we understand the problem to be variance in fees
+  forcing pools to keep large reserves.  For example, if a pool miner
+  controls 1% of hashrate and creates shares on a template with about
+  1,000 BTC in fees and 3 BTC in subsidy, they would be owed by their
+  pool about 10 BTC.  However, if the pool does mine that block and,
+  when they do mine a block, fees are back down to a fraction of the
+  block reward, the pool might only have 3 BTC total to split between
+  all of its miners, forcing it to pay from its reserves.  If that
+  happens too many times, the pool's reserves will be exhausted and it
+  will go out of business.  Pools address this in various ways,
+  including using various [proxies for actual fees][news304 fpps proxy].
+
+  Developer vnprc [described][vnprc ehash] the solution he's been
+  [building][hashpool] that focuses on ecash shares received in the
+  PPLNS payout scheme.  He thinks this could be especially useful for
+  launching new pools: right now, the first miner to join a pool suffers
+  the same high variance as solo mining, so typically the only people
+  who can start a pool are existing large miners or those willing to
+  rent significant hashrate.  However, with PPLNS ecash shares, vnprc
+  thinks a pool could launch as a client of a larger pool, so even the
+  first miner to join the new pool would receive lower variance than
+  solo mining.  The intermediate pool could then sell the ecash shares
+  it earns to finance whatever payout scheme it chooses to pay
+  its miners.  Once the intermediate pool acquired a
+  significant amount of hashrate, it would also have leverage for
+  negotiating with larger pools about creating alternative block
+  templates that suit its miners.
+
+- **Offchain DLCs:** developer conduition [posted][conduition offchain]
+  to the DLC-dev mailing list about a contract protocol that allows an
+  offchain spend of the funding transaction signed by both parties to
+  create multiple [DLCs][topic dlc].  After the offchain DLC has settled
+  (e.g., all required oracle signatures have been obtained), a new
+  offchain spend can be signed by both parties to reallocate funds
```

Comment:
```suggestion
  (e.g., all required oracle signatures have been obtained), both parties can sign a new
  offchain spend to reallocate funds
```
I'm not very much against passive voice (I know it's common in technical writing), but just before this, we have "been obtained."